ckanext/ontario_theme/templates/internal/snippets/facet_list.html

```diff
@@ -47,15 +48,14 @@ <h4 class="ontario-h5">
 
       <p class="module-footer">
         {% if h.get_param_int('_%s_limit' % name) %}
-          {% if h.has_more_facets(name, c.search_facets) %}
-            <a href="{{ h.remove_url_param('_%s_limit' % name, replace=0, extras=extras, alternative_url=alternative_url) }}"
-               role="button"
-               class="facets-show-more ontario-button ontario-button--tertiary">{{ _('Show more') }}</a>
+          {% if items|length > 5 %}
```

Comment:
Reinstate the original condition `{% if h.has_more_facets(name, search_facets or c.search_facets) %}` to use the limit as specified by `search.facets.default`, rather than hard-coding `5` here.

Reply:
You will have to figure out why line 26 `{% with items = items or h.get_facet_items_dict(name) %}` doesn't return all items, only the number defined in `search.facets.default`.